GDJS/Runtime/runtimegame.ts

```diff
@@ -1353,21 +1353,25 @@ namespace gdjs {
       const syncData: GameNetworkSyncData = {
         var: this._variables.getNetworkSyncData(syncOptions),
         ss: this._sceneStack.getNetworkSyncData(syncOptions) || undefined,
+        sm: this.getSoundManager().getNetworkSyncData() || undefined,
       };
-
       const extensionsVariablesSyncData = {};
       this._variablesByExtensionName.forEach((variables, extensionName) => {
         const extensionVariablesSyncData =
           variables.getNetworkSyncData(syncOptions);
         // If there is no variables to sync, don't include the extension in the sync data.
-        if (extensionVariablesSyncData.length) {
+        if (
+          extensionVariablesSyncData.length &&
+          !syncOptions.forceSyncEverything
```

Comment:
This logic is wrong. you don't need to check on `!syncOptions.forceSyncEverything` you always sync this